.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     working_directory: ~/scalar-jepsen
 
     docker:
-      - image: circleci/clojure:openjdk-8-lein-2.9.1
+      - image: circleci/clojure:openjdk-11-lein-2.9.8
```

Comment:
Does Jepsen 0.3.1 only supports `jdk 11` or later?

Reply:
Ah, I just took a look at this:
https://github.com/scalar-labs/verification/pull/55

I see Jepsen requires Java 11 or later. Thank you.